ci/eval/compare/default.nix

```diff
@@ -122,7 +122,7 @@ let
 in
 runCommand "compare"
   {
-    nativeBuildInputs = [
+    nativeBuildInputs = map lib.getBin [
```

Comment:
These kinda optimizations should probably be accompanied with a comment.

Same applies to other cases of `map getBin`.